src/pages/book/publish.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from "react";
```

Comment:
GENERAL COMMENTS:
- This page is not very mobile friendly. Maybe add a ticket for it on JIRA to fix the mobile response?
- Maybe follow up with Jordan or someone on whether they want a header on this page? (I.e. like a mockup a rectangle with the title of the page and then the books under the rectangle)